git.zsh

```diff
@@ -60,3 +65,36 @@ _fzf_complete_git_tabularize() {
         }
     '
 }
+
+_fzf_complete_git-add() {
+    _fzf_complete '--ansi' "$@" < <(git status --porcelain=v1 | awk \
+        -v green="$(tput setaf 2)" \
+        -v red="$(tput setaf 1)" \
+        -v reset="$(tput sgr0)" "
+        $fzf_colorize_git_status
```

Comment:
This looks nicer as `$0` is treated as a part of a string. Rename `y` to something pretty as well.
```zsh
        -v reset="$(tput sgr0)" '
        '"$fzf_colorize_git_status"'
        /^.[^ ]/ {
            y = substr($0, 2, 1)
            print colorize_git_status(green, red, reset)
        }
        ")

Reply:
Line with `y` is mistake, so I delete it.